gulpfile.js

```diff
@@ -160,21 +102,95 @@ gulp.task('generate-grids-topics', () => {
                     "igSelector": grid.igSelector
                 }
             }))
-            .pipe(gulp.dest(DOCFX_ARTICLES + grid.igPath));
+            .pipe(dest(DOCFX_ARTICLES + grid.igPath));
     }
-});
-
-gulp.task('build', [
-    'styles',
-    'cleanup',
-    'post-processor-configs',
-    'generate-grids-topics',
-    'build-site'
-]);
-
-gulp.task('build-travis', [
-    'styles',
-    'cleanup',
-    'post-processor-configs',
-    'generate-grids-topics'
-]);
\ No newline at end of file
+     setTimeout( () => eventEmitter.emit("finish"), 3000);
+     return eventEmitter;
```

Comment:
There is no need for the eventEmmitter and the setTimeout. The [watch ](https://gulpjs.com/docs/en/api/watch) func has an option args which allows you to pass a delay for the overlapped processes.

```suggestion
```

Reply:
Done